extensions/infinispan-cache/deployment/src/test/java/io/quarkus/cache/infinispan/InfinispanCacheTest.java

```diff
@@ -0,0 +1,341 @@
+package io.quarkus.cache.infinispan;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.Callable;
+import java.util.concurrent.CyclicBarrier;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.concurrent.SynchronousQueue;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import jakarta.inject.Inject;
+
+import org.infinispan.client.hotrod.RemoteCache;
+import org.infinispan.commons.jdkspecific.ThreadCreator;
+import org.infinispan.commons.util.NullValue;
+import org.jetbrains.annotations.NotNull;
```

Comment:
I know that IntelliJ adds these automatically, but we don;t really want them, so please make sure to remove them.

Reply:
oh did not see that thanks!!